protected/humhub/modules/admin/views/group/reassign-already-running.php

```diff
@@ -0,0 +1,23 @@
+<?php
```

Comment:
Can be removed, when working with statusMessages.

Reply:
Done!